core-feature-pack/src/main/resources/content/bin/standalone.sh

```diff
@@ -262,6 +262,9 @@ if [ "$PRESERVE_JAVA_OPTS" != "true" ]; then
         fi
     fi
 
+    # Set Java 9+ variable which will be used later
+    setJava9_plus
+
     if [ "$GC_LOG" = "true" ]; then
         # Enable rotating GC logs if the JVM supports it and GC logs are not already enabled
         NO_GC_LOG_ROTATE=`echo $JAVA_OPTS | $GREP "\-verbose:gc"`
```

Comment:
`verbose:gc` should not be used for gc-log detection, because `verbose:gc` is not used in JDK11+ ... right?

Reply:
As I understand it this is true that we don't need or want the `verbose:gc`.